app/src/pages/organization/organizationProjectsPage/projectsListTable/projectsListTable.scss

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.projects-list-table {
+  margin: 16px auto;
+  padding: 0 32px;
+
+  & > div:hover .date {
```

Comment:
Can we handle it without using `div` tag selector here?
Once the tag will be changed in the `ui-kit`, this behavior will be broken.

The general suggestion: use `class` selectors instead of `tag` selectors, especially while working with external modules.

Reply:
will add customization for table rows and header with adding custom classnames to avoid using directly tags,
will keep in mind to update component (there is story to add priority comlumn, costomize column sorting) and selector